### PR TITLE
Bug9273- change for reducing width and height for tree chart such that it fits the doc site

### PR DESCRIPTION
**Pull request by ankityadav4**
## Previous Behavior
 Tree chart not fitting in doc site

## New Behavior
 Adding new examples for Tree chart to be added in doc site.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: change/@fluentui-react-charting-fdcd995e-d8b9-4952-a4a2-2674d8757d49.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-fdcd995e-d8b9-4952-a4a2-2674d8757d49.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29370&commitId=443c197ea4887f170b933409f6e47eb13919325a&env=prod&clientType=FLUENTUI)


  <details><summary><b>MediaTile</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with nameplate hidden until hover.hover.chromium.png|175| Changed|

</details>


  <details><summary><b>react-charting-TreeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|115228| Changed|
|react-charting-TreeChart.Basic.default.chromium.png|91059| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|90986| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|109565| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|91288| Changed|
|react-charting-TreeChart.Compact.default.chromium.png|91385| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/TreeChart/TreeChart.base.tsx [hunk 2]
```diff
@@ -477,7 +477,7 @@ export class TreeChartBase extends React.Component<ITreeProps, ITreeState> {
 
   constructor(props: ITreeProps) {
     super(props);
-    this._margin = { top: 30, right: 20, bottom: 30, left: 50 };
+    this._margin = { top: 30, right: 130, bottom: 30, left: 50 };
     this._width = this.props.width || 1500;
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/TreeChart/TreeChart.base.tsx:481]: Will it help if we add margin as a prop as we have in other charts.
Right margin of 130 might make it work on docsite but leave too much empty space in actual usage

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by AtishayMsft — APPROVED** [review]

**Comment by AtishayMsft** [comment]
This is new change that has got introduced with my yesterday's PR and causing the build to fail.

ERR! Warning: You have changed the public API signature for this project. Please copy the file "temp/react-charting.api.md" to "etc/react-charting.api.md", or perform a local build (which does this automatically). See the Git repo documentation for more info.
ERR! [3:37:08 PM] x Error detected while running 'api-extractor:default'
ERR! [3:37:08 PM] x ------------------------------------
ERR! [3:37:08 PM] x Error: The public API file is out of date. Please run the API snapshot and commit the updated API file.
ERR!     at apiExtractorVerify (/mnt/vss/_work/1/s/node_modules/just-scripts/src/tasks/apiExtractorTask.ts:72:15)
ERR!     at _wrapFunction (/mnt/vss/_work/1/s/node_modules/just-task/lib/wrapTask.js:15:36)
ERR!     at taskWrapper (/mnt/vss/_work/1/s/node_modules/undertaker/lib/set-task.js:13:15)
ERR!     at bound (node:domain:433:15)
ERR!     at runBound (node:domain:444:12)
ERR!     at asyncRunner (/mnt/vss/_work/1/s/node_modules/async-done/index.js:55:18)
ERR!     at processTicksAndRejections (node:internal/process/task_queues:78:11)
ERR! [3:37:08 PM] x ------------------------------------
ERR! [3:37:08 PM] x finished 'api-extractor' in 4.54s with errors

**Review by AtishayMsft — COMMENTED** [review]
Reset review

**Comment by ankityadav4** [comment]
> Reset review



> This is new change that has got introduced with my yesterday's PR and causing the build to fail.
> 
> ERR! Warning: You have changed the public API signature for this project. Please copy the file "temp/react-charting.api.md" to "etc/react-charting.api.md", or perform a local build (which does this automatically). See the Git repo documentation for more info. ERR! [3:37:08 PM] x Error detected while running 'api-extractor:default' ERR! [3:37:08 PM] x ------------------------------------ ERR! [3:37:08 PM] x Error: The public API file is out of date. Please run the API snapshot and commit the updated API file. ERR! at apiExtractorVerify (/mnt/vss/_work/1/s/node_modules/just-scripts/src/tasks/apiExtractorTask.ts:72:15) ERR! at _wrapFunction (/mnt/vss/_work/1/s/node_modules/just-task/lib/wrapTask.js:15:36) ERR! at taskWrapper (/mnt/vss/_work/1/s/node_modules/undertaker/lib/set-task.js:13:15) ERR! at bound (node:domain:433:15) ERR! at runBound (node:domain:444:12) ERR! at asyncRunner (/mnt/vss/_work/1/s/node_modules/async-done/index.js:55:18) ERR! at processTicksAndRejections (node:internal/process/task_queues:78:11) ERR! [3:37:08 PM] x ------------------------------------ ERR! [3:37:08 PM] x finished 'api-extractor' in 4.54s with errors

I have merged the latest changes. I will check once.

File: change/@fluentui-react-charting-fdcd995e-d8b9-4952-a4a2-2674d8757d49.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-charting-fdcd995e-d8b9-4952-a4a2-2674d8757d49.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29370&commitId=7a6e12692afca8ea0f121f51fd668f9f5080b3cb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Pull request merged**